hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -158,6 +158,12 @@
   public static final String HBASE_MASTER_NORMALIZER_CLASS =
     "hbase.master.normalizer.class";
```

Comment:
If I want the new class as my normalizer, I'd set this?

Reply:
Yes. That's right.